### ISSUE TITLE
[FIX] Minimum page width

### ISSUE THREAD

**Issue by jtam360** [MyMood-Alexa/MyMood-Interface#23]
Set minimum width to prevent navigation bar from collapsing if user resizes the screen.

**Comment by jtam360** [comment]
I tried adding in css to set the min-width across all pages, but the layout, specifically the navbar, would still collapse even when the right portion of the navbar is off screen like so:
![non-collapsed navbar.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5acbd086ff8fbd00122ac6dc/125516c66e82c728ace21e0d46e69b946e818ca583e8af50a159b09a285025723b46ef696097ec22ba416f43554d4bfd120a0b40adf08c22a3e734778a5c0ee4d66a0d76936b03b0aac04ee155e57dd205d0f4ae805ec1ef3724e288874c3a1d011c8df6553db2d4.png)

Then I tried preventing the navbar from collapsing at all but then the navbar would get squished together as seen here:
![with collapse but squished.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5acbd086ff8fbd00122ac6dc/125516c66e82c728ace21e0d46ff9d8e2bc280a683e5be53b758b0963c5267603811aa387880e76af3006b1300051bff054a5556f9a8da79f2b26d798f550ebcd96a5f74c33416abea804ae913a632c91edbe2e3d7158abd7f37ed998e48341f001b81fc5936a5984dc05ff459ac.png)

Along the same lines, setting the navbar to a fixed-top prevents it from collapsing, however it messes with the rest of the html so it wasn't a good solution.

We cannot control the size of the browser window for users, so we can't set a minimum width for the canvas itself, we can only set min-width and min-height properties to control the page layout instead. It's also not possible to restrict the user from resizing their browser. The most we can do with the browser is change the size of the browser window when a user accesses the web page which isn't what we want at all. 

My final solution sets the minimum width along with preventing the navbar from collapsing which almost is what I was going for. The only "issue" there could be is that the navbar starts to be hidden off screen at the right when the screen size gets too small. Final solution:
![final navbar.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/5acbd086ff8fbd00122ac6dc/125516c66e82c728ace21e0d46ee9d94228ec3a78effbd42b61fbcd6395428792c07b7692ac7b629bc407f550e564ba544095743fcf98c22a1b16e72df0a19f1c32f0e2a92624cf5f2d75dad0ba8218c4584b3bf824fc4ec3e20e48b8f462c53.png)
